run_benchmarks.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 sbt "jmh/jmh:run -rf json -i 15 -wi 10 -f3 -jvmArgsAppend -XX:MaxInlineLevel=20 \
  -jvmArgsAppend -Xmx4g -jvmArgsAppend -XX:MaxInlineSize=99 \
+ -prof jmh.extras.JFR:dir=/tmp/filo-jmh \
```

Comment:
Is the idea here so we can always get JFR recording out?

Reply:
Correct